test/BUILD.gn

```diff
@@ -920,6 +922,7 @@ test("brave_browser_tests") {
     "//brave/components/url_sanitizer/browser",
     "//brave/components/web_discovery/buildflags",
     "//brave/components/webcompat/content/browser",
+    "//brave/components/webcompat/content/browser",
```

Comment:
is this a dep added to browser tests? That seems odd?

Reply:
fixed